### DISCUSSION TITLE
How to Setup UPP Grid_Mapping Parameter

### DISCUSSION THREAD

**Discussion by ShawnCebulaNOAA** [ufs-community/ufs-weather-model#1611] [Q&A]
Unable to delete discussion thread - posted in the incorrect repo, link to correct thread: https://github.com/NOAA-EMC/UPP/discussions/635

**Comment by natalie-perlin** [comment]
@jkbk2004 - FYI -
The issue could likely be closed, as it was moved to another repository/discussion that was more appropriate for the topic.
